src/lib/shared/ui/snackbar/snackbar-queue.svelte

```diff
@@ -0,0 +1,10 @@
+<script lang="ts">
+  import { snackbarQueue } from './store';
+  import Snackbar from './snackbar.svelte';
+
+  $: activeSnackbar = $snackbarQueue[0] || null;
+</script>
+
+{#if activeSnackbar}
+  <Snackbar options={activeSnackbar} />
```

Comment:
suggestion (non-blocking): I'd really love if there was some kind of transition here

Reply:
We'll have an entire sprint dedicated to animation and eye candy, I think it's best if we wait until then